devito/dse/rewriters.py

```diff
@@ -406,6 +408,40 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
         return cluster.rebuild(processed)
 
+    @dse_pass
+    def _skewing(self, cluster, template, **kwargs):
+        """
+        Skew the accesses along the time Dimension.
+        """
+
+        MAX_SKEW_FACTOR = 8
+        """
+        The maximum skew factor applied for the skewing pass.
```

Comment:
Btw, why should there be a maximum skew factor? 
do you mean that in certain cases, for certain numbers of time iterations and values of skew factors, there might be runtime segfaults due to illegal accesses?

Reply:
I also think this docstring would benefit from a simple example -- given a certain input expression, show the effect of skewing